.gitlab/e2e_testing_deploy/e2e_deploy.yml

```diff
@@ -222,3 +225,20 @@ deploy_windows_testing-a7-fips:
       $OMNIBUS_PACKAGE_DIR/pipeline-$CI_PIPELINE_ID s3://$WIN_S3_BUCKET/$WINDOWS_TESTING_S3_BUCKET
       --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
       full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
+
+# Uncomment this to add a job to export the testing public GPG key
+# Needed after rotating the testing GPG key
+# export_testing_public_key:
+#   rules:
+#     - !reference [.manual]
+#   needs: []
+#   stage: e2e_deploy
+#   image: registry.ddbuild.io/ci/datadog-agent-buildimages/gitlab_agent_deploy$CI_IMAGE_GITLAB_AGENT_DEPLOY_SUFFIX:$CI_IMAGE_GITLAB_AGENT_DEPLOY
+#   tags: ["arch:amd64"]
+#   script:
+#     - *setup_signing
+#     - /usr/bin/gpg-vault export-public-key $GPG_TEST_KEY_ID > gpg-testing-public-key.asc
+#   artifacts:
+#     paths:
+#       - gpg-testing-public-key.asc
+#     expire_in: 1 day
```

Comment:
Do we want to push a commented out block to the mainline? Couldn't we mark the job as `manual` instead?

Reply:
That was discussed in https://github.com/DataDog/datadog-agent/pull/36943#discussion_r2239827440 - I want the definition to be somewhere in the tree for the day we need it, but not having it active means one less job in the very long list of jobs in this pipeline.

Reply:
Sounds good to me

Reply:
Done in https://github.com/DataDog/datadog-agent/pull/36943/commits/7facf5010ff1b403ec2c4a9c0bfd235bab11f8f4

Reply:
Indeed, wrong comment thread. https://github.com/DataDog/datadog-agent/pull/36943/commits/8dfe3ade893852431c415742520fa01ab5dabc19 addresses this one.